doc/source/whatsnew/v0.23.0.txt

```diff
@@ -462,3 +462,4 @@ Other
 
 - Improved error message when attempting to use a Python keyword as an identifier in a ``numexpr`` backed query (:issue:`18221`)
 - :func:`Timestamp.replace` will now handle Daylight Savings transitions gracefully (:issue:`18319`)
+- :func:`DataFrame.info()` now shows line numbers for column summary (:issue:`17304`)
```

Comment:
this will need a sub-section for this to show as this is a major change to output formatting

Reply:
put this in new features